src/transformers/data/processors/squad.py

```diff
@@ -123,9 +123,11 @@ def squad_convert_example_to_features(example, max_seq_length, doc_stride, max_q
     truncated_query = tokenizer.encode(
         example.question_text, add_special_tokens=False, truncation=True, max_length=max_query_length
     )
+
+    tokenizer_type = type(tokenizer).__name__.replace("Tokenizer", "").lower()
```

Comment:
comment here would help! May also apply to GPT2 (which roberta and bart inherit from)